com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -336,12 +332,15 @@ internal void InitializeVars()
 
             {
                 // Create index map for channels
-                Dictionary<string, int> firstLevelIndex = new Dictionary<string, int>();
+                Dictionary<byte, int> firstLevelIndex = new Dictionary<byte, int>();
                 int secondLevelCounter = 0;
 
                 for (int i = 0; i < networkedVarFields.Count; i++)
                 {
-                    string channel = networkedVarFields[i].GetChannel(); // Cache this here. Some developers are stupid. You don't know what shit they will do in their methods
+                    // this could be cleaner.  The GetChannel() methods look for the SendChannel string channel name
+                    //  from the settings file.
+                    string channelName = networkedVarFields[i].GetChannel(); // Cache this here. Some developers are stupid. You don't know what shit they will do in their methods
```

Comment:
I would tone down the comment (yeah, I see it was there before), while we're touching this file. "Cache the channel name. Some developers might happen to mess with the channel name" is more aligned with our values.

LOL at original comment, though. Tempted to `git blame` to see who wrote that.

Reply:
I thought about the same thing but thought @TwoTenPvP might want to weigh in :)

Reply:
Ok, put in a friendlier / hopefully clearer comment